config.js

```diff
@@ -46,6 +46,7 @@ exports.updateFromArgv = function() {
         .describe('rejectnewmsg', 'Adds a customized reject message included in quote yourmessage quote')
         .default('rejectnewmsg', 'Currently, this bot is not accepting games, try again later ')
         .describe('rejectnewfile', 'Reject new challenges if file exists (checked each time, can use for load-balancing)')
+        .describe('rejectnewtime', 'Reject new challenges if local time of the day is later than specified DD:hh:mm (number of days to add, hour of the day, minute of the day')
```

Comment:
can you make DD: optional?

Reply:
fixed, now there is only hh:mm as you suggested on discord!
thanks!

Reply:
Should it say something like "number of days to add (optional)"?

Reply:
yeah thats a great idea, i didnt want to make pr more complicated but it may actually make it clearer!
will commit this a bit later :)